src/base/buffer.lisp

```diff
@@ -30,6 +30,10 @@
     :initarg :%enable-undo-p
     :accessor buffer-%enable-undo-p
     :type boolean)
+   (references 
+    :initform (make-hash-table :test 'equal)
+    :reader buffer-references
+    :accessor buffer-references )
```

Comment:
How about using `buffer-value`?

For example, define the following function where this is needed.
```common-lisp
(defun buffer-references (buffer)
  (buffer-value buffer 'references))
(defun (setf buffer-references) (value buffer)
  (setf (buffer-value buffer 'references) value))
```

Reply:
Oh I see, makes sense, I'll add it :+1: